deployments/gpu_plugin/overlays/nfd_labeled_nodes/add-nodeselector-intel-gpu.yaml

```diff
@@ -7,3 +7,4 @@ spec:
     spec:
       nodeSelector:
         feature.node.kubernetes.io/pci-0300_8086.present: "true"
+        feature.node.kubernetes.io/pci-0380_8086.present: "true"
```

Comment:
Doesn't this mean a node has to have both labels?
https://kubernetes.io/docs/concepts/scheduling-eviction/assign-pod-node/#nodeselector

Reply:
this triggers a question that should we use `intel.feature.node.kubernetes.io/gpu: "true"` here because of:
https://github.com/intel/intel-device-plugins-for-kubernetes/blob/3c948cc106914c8e7149b6b5e74e05490f4e8a7f/deployments/nfd/overlays/node-feature-rules/node-feature-rules.yaml#L52-L62